www/src/components/site-metadata.js

```diff
@@ -24,7 +24,7 @@ const SiteMetadata = ({ pathname, locale }) => {
   return (
     <Helmet defer={false} defaultTitle={title} titleTemplate={`%s | ${title}`}>
       <html lang={locale} />
-      <link rel="canonical" href={`${siteUrl}${pathname}`} />
+      <link rel="alternate" href={`${siteUrl}${pathname}`} />
```

Comment:
Pretty sure canonical isn't meant to be removed here. Rather that the locales are queried and alternate links are generated for each locale.

Reply:
OK. i work on it.

Reply:
Right. See the "with `hreflang`" section of this page: http://microformats.org/wiki/rel-alternate